src/main/resources/liquibase/changelog-1.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet id="1" author="Robin Rytkönen">
+        <comment>Add player table</comment>
+        <createTable tableName="Player">
+            <column name="Id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="playerId"/>
+            </column>
+            <column name="Name" type="VARCHAR(255)"/>
+            <column name="Email" type="VARCHAR(255)"/>
+            <column name="Balance" type="BIGINT"/>
+        </createTable>
+    </changeSet>
```

Comment:
Well done! If you previously have run it changeSet id="2" and now you are removing it from here you also need to clean up DATABASECHANGELOG table

Reply:
done!